index.js

```diff
@@ -89,12 +89,15 @@ module.exports = class GameActivityToggle extends Plugin {
 
                 r.props.children.unshift(React.createElement(r.props.children[0].type, {
                     icon: () => showCurrentGame ? enabledIcon('20') : disabledIcon('20'),
+                    'aria-label': `${showCurrentGame ? 'Hide' : 'Show'} Game Activity`,
+                    'aria-checked': `${showCurrentGame ? 'false' : 'true'}`,
+                    role: "switch",
                     onClick: () => {
                         showCurrentGame = !showCurrentGame
                         settings.updateRemoteSettings({ showCurrentGame })
                         forceUpdateElement('.' + classes.container, true)
                     },
-                    tooltipText: Messages.SHOW_CURRENT_GAME
+                    tooltipText: `${showCurrentGame ? 'Hide' : 'Show'}`
```

Comment:
`tooltipText` is used for tooltip text, not `aria-label`, so it should show what you want to hide or show
```suggestion
                    tooltipText: `${showCurrentGame ? 'Hide' : 'Show'} Game Activity`
```